torchao/utils.py

```diff
@@ -5,6 +5,8 @@
 from math import gcd
 from packaging import version
 import torch.nn.utils.parametrize as parametrize
+from torch.ao.quantization.utils import _assert_and_get_unique_device
```

Comment:
I can't seem to find this function on my end where is it defined?

```
(base) [marksaroufim@devgpu003.cco3 ~/ao (main)]$ grep -r "_assert_and_get_unique_device" .
```

Reply:
I'm using this function from https://github.com/pytorch/pytorch/blob/8a2fed7e6ab4cb6a97d92f6ca14f257370ec3a92/torch/ao/quantization/utils.py#L711